nuxt.config.ts

```diff
@@ -28,6 +32,7 @@ export default defineNuxtConfig({
         ? "ffdYLviQze3kzomaINXNk6NwpY9LlXcw"
         : "zbHWEXPUfXm0K6C7HbegwB5ewDEC8o1H",
       isAnalyticsEnabled: true,
+      siteUrl: process.env.NUXT_PUBLIC_SITE_URL || "https://qiskit.org",
```

Comment:
As far as I know, we don't have a environment variable called `NUXT_PUBLIC_SITE_URL`, is there something I'm missing?

Reply:
I was just following [this guide](https://unhead-schema-org.harlanzw.com/integrations/nuxt/module#2-configure-the-module) and it had it as a recommended guideline, but there's no further reason

Reply:
ok, i see what you mean, let me correct that.